dnnlib/comet_utils.py

```diff
@@ -0,0 +1,17 @@
+from comet_ml import Experiment
+import tensorflow as tf
+import json
+
+class CometLogger():
+    def __init__(self):
+            self.experiment = Experiment()
+
+    #Logging parameters
+    def log_parameters(self,hyper_parameters):
```

Comment:
```suggestion
    def log_parameters(self, hyperparameters):
```

Use pep8: space after comma; we spell without hyphen (or just "parameters" for short)

Reply:
Removed this function!